dashboard/app/mailers/ops_mailer.rb

```diff
@@ -34,9 +34,9 @@ def workshop_reminder(workshop, recipient)
     @workshop = workshop
     @recipient = recipient
 
-    subject = "Important: Your #{@workshop.phase_info[:long_name]} workshop is coming up in
```

Comment:
Why handle the @workshop.phase_info nil check differently for :long_name and :prerequisite_name? Should we generalize this, e.g. initializing phase_info to an empty hash so it's never null, or adding a safe lookup method?